staff/templates/staff/org_detail.html

```diff
@@ -126,6 +154,7 @@ <h2 class="h3">Projects</h2>
 {% endblock staff_content %}
 
 {% block extra_js %}
+<script type="text/javascript" src="{% static 'vendor/htmx.js' %}"></script>
```

Comment:
Could we use the minified version here instead?

```suggestion
<script type="text/javascript" src="{% static 'vendor/htmx.min.js' %}"></script>
```